docs/wiki/get-started/Installation.md

```diff
@@ -0,0 +1,17 @@
+## `pip`
+
+We ship binary wheels to install `csp`  on MacOS and Linux via `pip`:
+
+```bash
+pip install csp
+```
+
+## `conda` (coming soon)
```

Comment:
out of date, available for linux on conda (might as well say Mac too since that will happen very shortly)